api/v2/serializers/details/resource_request_actions.py

```diff
@@ -39,17 +39,20 @@ def create(self, validated_data):
         user = validated_data.get('approved_by')
         resource_request = validated_data.get('resource_request')
         identity = validated_data.get('identity')
-        data = {
-            'identity': identity.id,
-            'resource_request': resource_request.id,
-            'approved_by': user.username,
-            'quota': quota.id
-        }
-        event_serializer = QuotaAssignedByResourceRequestSerializer(data=data)
-        event_serializer.is_valid(raise_exception=True)
 
-        # Synchronous call to EventTable -> Set Quota for Identity's CloudProvider -> Save the Quota to Identity
```

Comment:
Might want to move this comment with the block.

Reply:
I nuked it, because its just saying what the code does.